### ISSUE TITLE
Re-enable image/video minatures rather than logos in timeline

### ISSUE THREAD

**Issue by keunes** [NUKnightLab/TimelineJS3#197]
In TimelineJS V2, in the timeline itself (i.e. the place with the overview, not the individual slide) each item got a miniature view of the image (or still of the video) of that slide. I think it's more valuable than having the logo in that overview (as currently is the case with V3). It is, after all, about the content, the story, and not the form (i.e. the platform/medium).
So as a feature request (sorry, I can't apply the label to this ticket), I would propose to either implement miniatures as standard, or let users choose whether they want logos or miniatures.

**Comment by JoeGermuska** [comment]
This is a good idea, but the path to this is a bit more complicated than I had guessed at first. Deferring work on this until after the 'cutover' milestone.

**Comment by thelinuxlich** [comment]
By the way it would be nice if I could customize that icon with a option inside the event item

**Comment by thelinuxlich** [comment]
Oh, I see there is a thumbnail property, but the media url is required, thus invalidating my use case. I just want a thumbnail to add *context* to the information presented on the slide text

**Comment by JoeGermuska** [comment]
@thelinuxlich Not sure what you mean. The media URL is *not* a required attribute of an event.

**Comment by thelinuxlich** [comment]
If you pass a media object, according to the docs, the url is required

**Comment by JoeGermuska** [comment]
Ah, I see. I'm OK with moving `thumbnail` to be a property of the event and not the media object. @zachwise any strong opinion?

**Comment by zachwise** [comment]
It's a strange use-case. A thumbnail by definition is a "a very small or concise description, representation, or summary." which works for media. The user clicks on a small representation of a media object and then sees that media object. In this case, if the user clicks on a media representation and then gets shown text instead it would feel disconnected. If we go this route, we would need to change the language to `icon` instead of thumbnail.

**Comment by JoeGermuska** [comment]
that's fair. And also probably unduly complicated unless we hear more calls for this feature. Still leaving this issue open for the larger issue of making the thumbnails work more like they did in TimelineJS2

**Comment by thelinuxlich** [comment]
Yeah, I'm seeing it more as a icon to represent the type of information I'm presenting on the slide
